yarn.lock

```diff
@@ -14173,11 +14178,35 @@ snack-babel-standalone@^2.2.2:
   resolved "https://registry.yarnpkg.com/snack-babel-standalone/-/snack-babel-standalone-2.2.2.tgz#2154855c482c112036261171ed32f7892102be60"
   integrity sha512-kYYfIsktDfJeYrByF184YO/x55U7rPzfXThsownEGCNBn8d1xZ8AD7NyuLS5aA1n2rk9yKQ1CZxIRHwoNkootQ==
 
+snack-content@~1.3.0:
```

Comment:
Hmm, these shouldn't change in the lockfile

Reply:
I think these are unrelated TBH, I tested deleting my `node_modules` and running `yarn install` from `main` and `yarn.lock` would still change

Reply:
Oh wait, what happens when you run these commands in order from the root of the project?

- `yarn install`
- `yarn build`
- `yarn install`

Reply:
Let's merge it, I can check before deploying.

Reply:
> Oh wait, what happens when you run these commands in order from the root of the project?
> 
> * `yarn install`
> * `yarn build`
> * `yarn install`

Same thing, unfortunately